pkg/localize/locales/en/cmd/cluster.en.toml

```
@@ -4,6 +4,9 @@ one = 'cluster'
 [cluster.cmd.shortDescription]
 one = 'View and perform operations on your Kubernetes or OpenShift cluster'
 
+[cluster.cmd.longDescription]
+one = 'Use these commands to connect and bind your services to Kubernetes or OpenShift applications. You can also check if the Kubernetes or OpenShift cluster has the required operators to perform this operation.'
```

```suggestion
one = 'Use these commands to connect and bind your services to Kubernetes or OpenShift applications. You can also check if the Kubernetes or OpenShift cluster has the required Operators to perform this operation.'
```
I know it's awkward, but "Operator" is always supposed to be capitalized in Kubernetes and OpenShift (even if it's not being used as a proper noun).